src/harrastuspassi/harrastuspassi/tests/test_hobby_api.py

```diff
@@ -130,6 +130,12 @@ def test_hobby_update(user_api_client, valid_hobby_data):
     assert response.status_code == 200
     hobby_obj = Hobby.objects.get(id=response.data['id'])
     assert hobby_obj.name == hobby_data['name']
+    # test put with cover_image missing from hobby data
```

Comment:
This comment reads a bit confusing, without knowing the story behind it. Maybe at least rephrase "missing" to "test without cover_image".